scripts/retort-sync.ps1

```
@@ -0,0 +1,144 @@
+        if (Test-Path $src) {
+            $dest = Join-Path $BackupDir $relPath
+            New-Item -ItemType Directory -Path $dest -Force | Out-Null
+            Copy-Item -Path "$src\*" -Destination $dest -Recurse -Force
+        }
+    }
+    $relBackup = $BackupDir.Replace("$ProjectRoot\", '')
+    Write-Host "[retort-sync] Backup written to: $relBackup"
+}
+
+# -- Apply sync --------------------------------------------------------------
+Write-Host ''
```

PSScriptAnalyzer (1.24.0)

[warning] File 'retort-sync.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)